### PR TITLE
Add logging URL to tenant attributes

### PR DESCRIPTION
**Pull request by stooke**
This PR adds the logging URL (to accompany the console and metrics URLs) to the tenant attributes.

After this is merged, the logging URL will be retrieved from the Auth service, and added to the endpoint /api/tenant.

Following this PR, a PR for the WIT service will consume the logging URL.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -67,6 +69,9 @@ import:
 - package: github.com/bitly/go-simplejson
   version: ^0.5.0
 - package: github.com/google/go-querystring
+- package: github.com/pilu/config
```
Comment (aslakknutsen) [inline glide.yaml]: technically not part of this PR

Comment (stooke) [reply glide.yaml]: Absolutely true.  That's from PR #539, but I can't build tenant without it.

**Comment by aslakknutsen** [comment]
[test]

File: token/service.go [hunk 2]
```diff
@@ -57,10 +57,7 @@ func (s *tokenService) ResolveTargetToken(ctx context.Context, target, token str
 	if err != nil {
 		return "", "", errors.Wrapf(err, "error while decoding the token for %s", target)
 	}
-	if externalToken.Username == nil {
```
Comment (aslakknutsen) [inline token/service.go]: ?

Comment (stooke) [reply token/service.go]: The Auth service changed its API contract to require Username on output.  Because of this, Goa changed the generated type from a pointer-to-string to an actual string.  Because it's an actual string (so never nil) and required (so never length 0) I took out the test and removed a later indirection.

If you wish I can add a test to ensure non-empty string; I just felt that was redundant.

Comment (aslakknutsen) [reply token/service.go]: Re-enable the check for empty string. We want to know specifically if it's "" instead of some random unknown failure later. Can't trust @alexeykazakov ;)

Comment (aslakknutsen) [reply token/service.go]: Unless @alexeykazakov can confirm it's always there now. :)

Comment (alexeykazakov) [reply token/service.go]: It's always there unless we have a bug in Auth :) So, don't trust me :)

Comment (alexeykazakov) [reply token/service.go]: To be clear. Username should be always there. It's why we changed it from optional to required. But trust no one! Auth can have bugs (hard to believe though). Check the output and report an error if it's not what you expected.

Comment (stooke) [reply token/service.go]: Test added.  Best part is the previous code had a bug in it: missing %s !

**Comment by fabric8cd** [comment]
@stooke snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-543-4`
